config/main.py

```diff
@@ -986,6 +988,129 @@ def load(filename, yes):
         log.log_info("'load' executing...")
         clicommon.run_command(command, display_cmd=True)
 
+@config.command('apply-patch')
+@click.argument('patch-file-path', type=str, required=True)
+@click.option('-f', '--format', type=click.Choice([e.name.lower() for e in ConfigFormat]),
```

Comment:
>format [](http://example.com/codeflow?start=23&length=6)

This option is not in design doc.

Reply:
This option is added to make it easy to use `SonicYang` format in the future once our internal services start interacting with SONiC using `SonicYang`.

Reply:
Better to update the design doc.